python/pyspark/mllib/tests.py

```diff
@@ -620,6 +621,42 @@ def test_right_number_of_results(self):
         self.assertEqual(len(chi), num_cols)
         self.assertIsNotNone(chi[1000])
 
+
+class Word2VecTests(PySparkTestCase):
+    def test_word2vec_setters(self):
+        data = [
+            ["I", "have", "a", "pen"],
+            ["I", "like", "soccer", "very", "much"],
+            ["I", "live", "in", "Tokyo"]
+        ]
+        model = None
+        try:
+            model = Word2Vec()\
+                .setVectorSize(2)\
+                .setLearningRate(0.01)\
+                .setNumPartitions(2)\
+                .setNumIterations(10)\
+                .setSeed(1024)\
+                .setMinCount(3)
+        except:
```

Comment:
This `try..except..` doesn't do any thing but hide possible exception messages. It is okay to have multiple exceptions and people will check the first one.